src/main/java/com/redhat/devtools/intellij/quarkus/javadoc/JavadocContentAccess.java

```diff
@@ -20,11 +20,16 @@
 
 public class JavadocContentAccess {
     private static Reader getHTMLContentReader(PsiMember member, boolean allowInherited, boolean useAttachedJavadoc) {
-        PsiDocComment doc = ((PsiDocCommentOwner) member).getDocComment();
+        PsiDocComment doc = null;
         PsiElement sourceMember = member.getNavigationElement();
         if (sourceMember instanceof PsiDocCommentOwner) {
             doc = ((PsiDocCommentOwner) sourceMember).getDocComment();
         }
+        if (doc == null) {
+            if (member instanceof PsiDocCommentOwner) {
```

Comment:
I will do this test at first and set the code PsiElement sourceMember = member.getNavigationElement();.. in an else

Reply:
I just kept the same logic as before

Reply:
done